### DISCUSSION TITLE
Clarification on bottles

### DISCUSSION THREAD

**Discussion by tmspzz** [Homebrew/discussions#315] [Everyday usage]
Hello, I'm looking for clarification around this PR https://github.com/Homebrew/brew/pull/8410 that was merged into core.

How are we supposed to use this feature in our own formula? 
In general, is there a way to have home-brew default to downloading from a GitHub release without opening tap?

**Comment by iMichka** [comment]
We made a blog post about the process: https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

And yes you will need to create a tap for this.

**Reply by tmspzz** [reply]
Got it, thanks for the clarification 👍  . I think it would be useful to explain usage in the docs too if possible, I was looking there.

**Reply by iMichka** [reply]
If someone finds time, it would be great to port the blog post to our doc :) I don't have that time, but anyone can pick this up.
